### ISSUE TITLE
How to detect history change?

### ISSUE THREAD

**Issue by juliadou** [WICG/portals#168]
In the latest chrome canary, whenever portal is activated, the change gets pushed into the history stack so user can use the browser back button to navigate back to the previous page. This is super nice.

But I am having trouble detecting the history change. Under Site A, I activated a portal pointing to Site B (url changed to Site B) and i pressed browser back button (it navigates back to Site A). Under Site B, i have window.onpopstate attached to an event handler but it is not triggered when browser back button is clicked. I am trying to activate the Site A portal under Site B under the onpopstate handler.

**Comment by jeremyroman** [comment]
`onpopstate` isn't generally fired on back button unless the previous history entry is one generated by `history.pushState`. More generally, sophisticated handling of back-navigation is still a work in progress (mainly by @kjmcnee), which issue #19 tracks. As we enter the holiday season it may be a little while before you see further progress in this area.

**Comment by domenic** [comment]
I think this issue is still valid. In particular the experience @juliadou describes is one I don't know how you would create using our current set of events.

**Comment by jeremyroman** [comment]
Yeah. Nothing's implemented here yet AFAIK, but I believe @kjmcnee has ideas about how to expose this event.
